api/.gitignore

```diff
@@ -0,0 +1,160 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
This file should go one folder up

Reply:
I think it makes more sense to keep it there, it's only the python stuff in this gitignore, so I think it belongs in the python server folder.